.github/workflows/intl-data-tests.yml

```diff
@@ -84,11 +90,49 @@ jobs:
         run: |
           [ -f src/Symfony/Component/Intl/Resources/data/locales/en.php ]
           [ ! -f src/Symfony/Component/Intl/Resources/data/locales/en.php.gz ]
-          [ -f src/Symfony/Component/Intl/Resources/data/transliterator/emoji/emoji-en.php ]
-          [ ! -f src/Symfony/Component/Intl/Resources/data/transliterator/emoji/emoji-en.php.gz ]
           src/Symfony/Component/Intl/Resources/bin/compress
           [ ! -f src/Symfony/Component/Intl/Resources/data/locales/en.php ]
           [ -f src/Symfony/Component/Intl/Resources/data/locales/en.php.gz ]
-          [ ! -f src/Symfony/Component/Intl/Resources/data/transliterator/emoji/emoji-en.php ]
-          [ -f src/Symfony/Component/Intl/Resources/data/transliterator/emoji/emoji-en.php.gz ]
           ./phpunit src/Symfony/Component/Intl
+
+  tests-emoji:
+    name: Emoji data
+    runs-on: Ubuntu-20.04
+    steps:
+      -   name: Checkout
+          uses: actions/checkout@v4
+
+      -   name: Setup PHP
+          uses: shivammathur/setup-php@v2
+          with:
+            coverage: "none"
+            extensions: "zip,intl"
+            ini-values: "memory_limit=-1"
+            php-version: "8.2"
+
+      -   name: Install dependencies
+          run: |
+            COMPOSER_HOME="$(composer config home)"
+            ([ -d "$COMPOSER_HOME" ] || mkdir "$COMPOSER_HOME") && cp .github/composer-config.json "$COMPOSER_HOME/config.json"
+            export COMPOSER_ROOT_VERSION=$(grep ' VERSION = ' src/Symfony/Component/HttpKernel/Kernel.php | grep -P -o '[0-9]+\.[0-9]+').x-dev
+            echo COMPOSER_ROOT_VERSION=$COMPOSER_ROOT_VERSION >> $GITHUB_ENV
+
+            echo "::group::composer update"
+            composer update --no-progress --ansi
+            echo "::endgroup::"
+
+            echo "::group::install phpunit"
+            ./phpunit install
+            echo "::endgroup::"
+
+      -   name: Run tests
+          run: ./phpunit src/Symfony/Component/Emoji
+
+      -   name: Test with compressed data
```

Comment:
let's move these as steps in the previous job, no need to repeat installing php et al.